dex/msgjson/types.go

```diff
@@ -1295,9 +1295,7 @@ type ConfigResult struct {
 	// by bond version.
 	BondExpiry uint64 `json:"DEV_bondExpiry"`
 
-	RegFees        map[string]*FeeAsset `json:"regFees"`
-	Fee            uint64               `json:"fee"`            // DEPRECATED
-	RegFeeConfirms uint16               `json:"regfeeconfirms"` // DEPRECATED
```

Comment:
Legacy clients will see a `Fee` of 0, and block registration (See `(*Core).Register`)